item-spec/item-spec.md

```diff
@@ -252,6 +218,53 @@ in the file).
 For examples of fields that this construct is recommended for, see the [section of STAC Best Practices](../best-practices.md#common-use-cases-of-additional-fields-for-assets)
 that talks about common use cases of additional fields for assets.
 
+## Media Types
```

Comment:
Wasn't there an issue to move this to a separate page anyway? This would be a good chance to move it...

Reply:
There was - #930 I closed it since there was no real consolidation to do, and I don't love specs where you have to keep looking at other pages...

Reply:
I still think it would make sense as it's a central part of all three specs and not really bound to items as you've realized yourself. Same for the rel types. At the moment you have to look at other pages anyway for Catalog and Collection, so it's only different for items.

Reply:
Ok, could you edit it in the way you want? Feel free to make commits on this branch directly. I wasn't sure where/how to make a shared page between the specs, but it seems like you have a way you want it done.

Reply:
No clear idea yet to be honest. Could be similar to common metadata? Or should me create for everything common (rel types, media types, common metadata)?

Reply:
Ok, just made the changes as discussed on the call. Moved most of the table to 'best practices', and kept just our custom (stac and cog) ones in this table.